.github/workflows/links.yml

```diff
@@ -26,7 +26,9 @@ jobs:
           # the whole of the project slug README since the links are
           # necessarily not real links - i.e. demos and to be filled by
           # cookiecutter values.
-          args: "--verbose --exclude https://forum.image.sc/ --exclude docs/pages --exclude-path '{{cookiecutter.project_slug}}/README.md'  -- ."
+          args:
+            "--verbose --exclude https://forum.image.sc/ --exclude docs/pages
+            --exclude-path '{{cookiecutter.project_slug}}/README.md'  -- ."
```

Comment:
No problem, but I'm surprised that this works. 🤷

Reply:
Yeah, I thought the same, but verified with [jless](https://jless.io)
<img width="1216" alt="image" src="https://github.com/UCL-ARC/python-tooling/assets/15052188/8ef0faf7-ac2a-481e-be70-45fda111376a">